FabLabKasse/faucardPayment/faucard.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, parent, amount):
         :type amount: Decimal float
```

Comment:
The `:type:` docs should be updated to match isinstance (for all changes to `isinstance`)